taskrunners/downloader/src/Connect.py

```diff
@@ -0,0 +1,11 @@
+from pymongo import MongoClient
+
+db = None
+
+def connect():
+    global db
+    domain = 'domain.com'
+    username = 'USERNAME'
+    password = 'PASSWORD'
+    authSource = 'user_table'
+    return MongoClient(domain, 27017, username=username, password=password, authSource=authSource).finlab_prod
```

Comment:
Is there a reason this is finlab_prod? The other was finlab_beta. Connect.py don't connect to the same db.

Reply:
Updated the other to finlab_prod.